.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rmf_visualization_schedule/jwt-cpp"]
```

Comment:
looks like this points to a commit in master branch, I would recommend to use a release tag instead.